services/QuillLMS/spec/models/activity_health_spec.rb

```diff
@@ -28,4 +28,28 @@
   it { should validate_numericality_of(:recent_plays).is_greater_than_or_equal_to(0)}
   it { should validate_inclusion_of(:avg_difficulty).in_range(0..5)}
 
+  context '#serializable_hash' do
```

Comment:
I would recommend using the rspec template of
`expect(____).to eq ____` over `assert_equals` since this communicates what is "actual" vs. "expected".  Here's a good [post](https://gist.github.com/myronmarston/5889730)